src/components/tiles/geometry/geometry-content.tsx

```diff
@@ -1641,6 +1646,21 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
       const coords = copyCoords(point.coords);
       const isPointDraggable = !this.props.readOnly && !point.getAttribute("fixed");
 
+      if (mode === "circle") {
+        // Either start a circle, or close the active circle using the clicked
```

Comment:
```suggestion
        // Either start a circle, or close the active circle using the clicked...
```
Don't leave me hanging like that -- I'm on the edge of my seat here.... 😉